sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -386,7 +386,7 @@ export class EventHubClient {
    * @param {(string|number)} partitionId Partition ID for which partition information is required.
    * @returns {Promise<PartitionProperties>} A promise that resoloves with EventHubPartitionRuntimeInformation.
    */
-  async getPartitionInformation(partitionId: string, cancellationToken?: Aborter): Promise<PartitionProperties> {
+  async getPartitionInformation(partitionId: string|number, cancellationToken?: Aborter): Promise<PartitionProperties> {
```

Comment:
The type for partitionId should be just `string`.
If a javascript user passes a number, then we can do the string conversion internally, but the officially type for partitionId should be just `string`

Reply:
On second thoughts, we can take care of this as part of #2660